app/scripts/views/mixins/inter-tab-signin-mixin.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'cocktail',
+  'lib/constants',
+  'lib/promise',
+  'views/mixins/inter-tab-channel-mixin'
+],
+function (Cocktail, Constants, p, InterTabChannelMixin) {
+  'use strict';
+
+  var EVENTS = Constants.EVENTS;
+
+  return Cocktail.mixin({
+    afterVisible: function () {
+      this.navigateToSignedInView = navigateToSignedInView.bind(this);
+      this.interTabOn(EVENTS.SIGNIN_SUCCESS, this.navigateToSignedInView);
+    },
+
+    onSignInSuccess: function (account) {
+      this.interTabOff(EVENTS.SIGNIN_SUCCESS, this.navigateToSignedInView);
```

Comment:
Based on my testing, yes it does and yes it is. Will double check `crosstab` to make sure I'm not being stupid though.